mautic/mautic
# [UX] Improve the explanation of the Manual Copy way to implement the form

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
This revision addresses the confusion surrounding the manual copy method for form implementation. 

It clarifies that while the platform generates the complete HTML code for self-hosting, subsequent alterations made within the platform do not automatically propagate to the live website. Users must manually update the HTML on the respective pages to reflect these changes. 

This distinction is crucial for users who manage forms outside the platform, ensuring they understand the need for manual intervention to keep their content current.


#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open Forms
3. Create a form
4. Salve and close the form
5. Click on Manual Copy
6. Read the description

![image](https://github.com/mautic/mautic/assets/116097999/fab6f347-398b-4669-9230-9a9b49681966)
